mindjunkies/accounts/management/commands/populate_db.py

```
@@ -0,0 +1,72 @@
+            )
+            profile, created = Profile.objects.get_or_create(
+                user=user,
+                defaults={
+                    'birthday': fake.date_of_birth(minimum_age=18, maximum_age=80),
+                    'bio': fake.paragraph(nb_sentences=3),
+                    'phone_number': fake.phone_number()[:15],
+                    'address': fake.address(),
+                    'avatar': None
+                }
+            )
+            self.stdout.write(f"Created user: {username} ({'Teacher' if is_teacher else 'Student'}) - Profile {'created' if created else 'already existed'}")
```

Ruff (0.8.2)

Line too long (157 > 119)

(E501)